runtime/src/assets.rs

```diff
@@ -11,6 +11,13 @@ lazy_static! {
     static ref ASSET_REGISTRY: Mutex<AssetRegistry> = Mutex::new(AssetRegistry::default());
 }
 
+/// The symbol of Lend Tokens is the symbol of their underlying currency,
+/// prefixed by `Q`. Example: `QDOT` is the `DOT` lend token symbol.
+const LEND_TOKEN_SYMBOL_PREFIX: char = 'Q';
+/// The name of Lend Tokens is the name of their underlying currency,
+/// prefixed by `Lend`. Example: `LendPolkadot` is the `Polkadot` lend token name.
```

Comment:
This is problematic if the underlying name contains spaces. Perhaps better would be `Lend(Polkadot)`. But I'd suggest seeing if we can just remove the `name()` first

Reply:
Removed it, it wasn't used